.gitignore

```diff
@@ -104,6 +104,7 @@ ENV/
 *~
 *.swp
 *.swo
+tags
```

Comment:
This IMO should got into global gitignore, the reason being the file name is customizable,

Reply:
I concur on that. That's something for your global gitignore, rather than something in this git repo.

Reply:
I agree. I have removed that. Thanks @sayanchowdhury and @edenemmanuel for your review :)